components/form-builder/app/edit/elements/lexical-editor/Editor.tsx

```diff
@@ -41,7 +39,6 @@ export const Editor = ({
     undefined
   );
 
-  const { status: formBuilderDebugEnabled } = useFlag("formBuilderDebug");
   const [showTreeView, setShowTreeView] = useState<boolean>(false);
```

Comment:
Noticed that this state isn't used in the Toolbar child anymore.  Could possibly be removed?